src/xGadgetron/cGadgetron/tests/CMakeLists.txt

```diff
@@ -35,30 +35,41 @@ if (DOWNLOAD_ZENODO_TEST_DATA)
                 "a692ba3a6c456d1ad49750717a9d5dda")
 endif()
 
-OPTION(RERUN_ISMRMRD_SHEPP_LOGAN "Generate testdata compatible with ISMRMRD Version" OFF)
+OPTION(RERUN_ISMRMRD_SHEPP_LOGAN "Generate testdata compatible with ISMRMRD Version" ON)
 find_program(EXE_ISMRMRD_SL NAMES ismrmrd_generate_cartesian_shepp_logan
                 NAMES_PER_DIR
                 PATHS ${CMAKE_INSTALL_PREFIX})
+if (EXE_ISMRMRD_SL STREQUAL "EXE_ISMRMRD_SL-NOTFOUND")
+  set (EXE_ISMRMRD_SL_FOUND OFF)
+else()
+  set (EXE_ISMRMRD_SL_FOUND ON)
+endif()
```

Comment:
The [docs](https://cmake.org/cmake/help/latest/command/find_program.html?highlight=find_program) say that 
> If the program is found the result is stored in the variable and the search will not be repeated unless the variable is cleared. If nothing is found, the result will be <VAR>-NOTFOUND.

I understand that in either case `EXE_ISMRMRD_SL` will exist and `if (EXE_ISMRMRD_SL)` will evaluate to true. 
However, we won't know what we want as if the program it's not found it will set to `EXE_ISMRMRD_SL-NOTFOUND`.

Reply:
https://cmake.org/cmake/help/latest/command/if.html says
> `if(<constant>)`
> True if the constant is 1, ON, YES, TRUE, Y, or a non-zero number. False if the constant is 0, OFF, NO, FALSE, N, IGNORE, NOTFOUND, the empty string, or ends in the suffix -NOTFOUND. 

which is why I thought `EXE_ISMRMRD_SL_FOUND` is equivalent to `EXE_ISMRMRD_SL` .

minor point of course

Reply:
done